com.microsoft.mrtk.input/Interactors/GazePinch/GazePinchInteractor.cs

```diff
@@ -199,17 +214,15 @@ private void ComputeAttachTransform(IXRSelectInteractable interactable)
             // Get the actual device/grab rotation. The controller transform is the aiming pose;
             // we must get the underlying grab rotation.
             // TODO: Replace with explicit binding to OpenXR grip pose when the standard is available.
-            if (xrController is ActionBasedController abController &&
-                abController.rotationAction.action?.activeControl?.device is TrackedDevice device)
+            if (rayPoseSource.TryGetPose(out Pose rayPose))
             {
-                rotationToApply = PlayspaceUtilities.ReferenceTransform.rotation * device.deviceRotation.ReadValue();
+                attachTransform.rotation = PlayspaceUtilities.ReferenceTransform.rotation * rayPose.rotation;
```

Comment:
Hmm...is this working? It seems like the rotation will be overridden down on line 254. We could probably leave this as `rotationToApply =` so the full pose is applied together.

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/be4aef1098b46fa34c6a1bfe04851b905b8bc440/com.microsoft.mrtk.input/Interactors/GazePinch/GazePinchInteractor.cs#L254

Reply:
Fixed